pkg/resources/resource_cluster.go

```diff
@@ -15,6 +15,12 @@ import (
 var clusterSchema = map[string]*schema.Schema{
 	"name":           NameSchema("cluster", true, true),
 	"ownership_role": OwnershipRole(),
+	"replication_factor": {
```

Comment:
Should `replication_factor` and `size` both include `RequiredWith` since you would need to set them together.